lang/notes/de.txt

```diff
@@ -1,13 +1,143 @@
-This file contains notes/discussion regarding translation. Look here if there's a term that's common in the game and you don't know how to translate.
+This file contains notes/discussion regarding translation for German, written
```

Comment:
Why do you format text manually?

Reply:
80 characters per line, it's a common convention.

Reply:
Take a look on other text file in repo, ```README_all_translators.txt``` for example. It doesn't follow your convention, because all text editors and github can do the proper wrapping automatically.

Your formatting makes a half of my screen useless:
![capture](https://user-images.githubusercontent.com/5421954/38765845-d5b39ea0-3fd1-11e8-8a2e-827848947441.PNG)

Reply:
And, BTW, text you have changed were not wrapped like you did, so I'll call this regression of formatting.